uswid/identity.py

```diff
@@ -304,6 +305,31 @@ def export_json(self) -> bytes:
         # success
         return json.dumps(root, indent=2).encode("utf-8")
 
+    def import_pkg_config(self, txt: str, fn=None) -> None:
```

Comment:
I think it would be more readable if 'fn' variable is called 'path' or 'filepath'.
I had to think twice, for what it stands for and whether or not it's just the filename or a path.

Reply:
I've used fn pretty much everywhere in uswid already. :/